public/static/partials/project_trigger_modal.html

```diff
@@ -36,11 +36,11 @@
                     <section layout="row" flex>
                         <md-input-container flex=50>
                             <label>Variant regex:</label>
-                            <input type="text" ng-model="trigger.variant_regex" placeholder="Only matching variants will trigger a build">
+                            <input type="text" ng-model="trigger.variant_regex" placeholder="Only matching variant identifiers will trigger">
```

Comment:
This feels like it should also say "names" instead of "identifiers", since that's what it says below, and identifiers is a little hard for people to understand.

Reply:
The identifier rather than the name is used here. If you have a different suggestion for wording though I can change it

Reply:
Oh, I see. That makes sense.